config.go

```diff
@@ -349,7 +349,11 @@ func (c *ConfigEntry) GetString(key string, defValue string) string {
 	s, ok := c.keyValues[key]
 
 	if ok {
-		return s
+		env := NewStringExpression("here", c.ConfigDir)
```

Comment:
Without stomping all over anyones toes, maybe use the new method I added in this patch?

https://github.com/ochinchina/supervisord/pull/29/files#diff-b4bda758a2aef091432646c354b4dc59R358